Presentation.Web/Controllers/API/V2/External/DataProcessingRegistrations/Mapping/DataProcessingRegistrationResponseMapper.cs

```diff
@@ -27,7 +27,7 @@ public DataProcessingRegistrationResponseDTO MapDataProcessingRegistrationDTO(Da
                 General = MapGeneral(dataProcessingRegistration),
                 SystemUsages = MapSystemUsages(dataProcessingRegistration),
                 Oversight = MapOversight(dataProcessingRegistration),
-                Roles = MapRoles(dataProcessingRegistration),
+                Roles = MapRoles(dataProcessingRegistration).ToList(),
```

Comment:
not needed...
![image](https://user-images.githubusercontent.com/13116889/162156399-0c671f87-7181-404b-a3e0-5d365c6ee36f.png)